core/src/main/scala/kafka/coordinator/GroupMetadataManager.scala

```diff
@@ -458,7 +462,7 @@ class GroupMetadataManager(val brokerId: Int,
           }
         }
 
-        val (groupOffsets, emptyGroupOffsets) = loadedOffsets
+        val (groupOffsets, emptyGroupOffsets) = loadedOffsets.toMap
```

Comment:
Yes, it makes mutable.Map loadedOffsets immutable, so you can safely call mapValues on it without any side effects which could be possibly done on loadedOffsets Map.